src/withAxiom.ts

```diff
@@ -101,13 +101,14 @@ function withAxiomNextApiHandler(handler: NextApiHandler): NextApiHandler {
   };
 }
 
-function withAxiomNextMiddleware(handler: NextMiddleware): NextMiddleware {
+function withAxiomNextEdgeFunction(handler: NextMiddleware): NextMiddleware {
   return async (req, ev) => {
     try {
       const res = await handler(req, ev);
       ev.waitUntil(log.flush());
       return res;
     } catch (error) {
+      console.log('Error in edge function', error);
```

Comment:
```suggestion
```

Not sure if we should `console.log` here, errors will appear twice with this: Unstructured (`console.log`) and structured (`log.error`).